### DISCUSSION TITLE
Advice on how to set up DDNS-Updater with Cloudflare

### DISCUSSION THREAD

**Discussion by Anos666** [qdm12/ddns-updater#779] [General]
I seem to keep getting the same error code referring to “Back-off restarting failed container ddns-updater in pod ddns-updater-5c457c465c-qlvmq_ix-ddns-updater(664b0d50-3d25-45c8-a2ca-77d4b4b91291)” the section in the parenthesis keeps updating to different things but the error ruffly stays the same, any suggestions on what I am doing wrong?

**Comment by qdm12** [comment]
Please share the container logs as to why it fails to start, otherwise I cannot help more.
